### PR TITLE
Revision changes for Alpha1-3

### PR DESCRIPTION
**Pull request by highlander1599**
Final changes for Alpha 1-3

### CODE REVIEW

**Comment by highlander1599** [comment]
Please don't merge. I accidentally used an old revision version. Should have checked that before. Sorry

**Comment by highlander1599** [comment]
Errors fixed, ready to merge.

**Comment by highlander1599** [comment]
Changelog for the changes so far

Flamer range increased to 4 tiles from 3 tiles, Inferno range increased to 5.5 from 4 tiles

Alpha 01
First base: Added the first MG damage technology
Second base: The repair turret moved to the base 3 artifact for Normal and lower difficulties, Flamer turret now prerequisite for the first Flamer damage upgrade
Third base: Artifact contains the second MG damage upgrade 
Range of the Flamer towers get decreased to 0.5 tiles lower than the MG range
Range of the Scavenger Flamer get decreased to the same range as the MG

Alpha 02: 
First base: Artifact contains the second Flamer damage upgrade which is the prerequisite for the first ROF and the range upgrade
NW base: Twin MG becomes the prerequisite of the third MG damage upgrade
The Scavenger get no Flamer range upgrade, Flamer range still decreased for Normal and lower difficulties;
for Hard and Insane difficulty the Flamer range is set to its original value, for Insane difficulty the range upgrade is added

Alpha 03:
The Heavy Machinegun is no longer the prerequisite of the third MG damage upgrade (moved to the Twin MG)
Scavenger Flamer is set back to its original range, no range upgrade for Normal and lower difficulties but for Hard and Insane difficulty

Alpha 04
One wall and one structure material upgrade for the Scavenger

File: script/campaign/cam1b.js [hunk 1]
```diff
@@ -85,9 +85,20 @@ function eventStartLevel()
 
 	camCompleteRequiredResearch(SCAVENGER_RES, 6);
 	camCompleteRequiredResearch(SCAVENGER_RES, 7);
-
+	if (difficulty === HARD)
+	{
+		completeResearch("R-Wpn-Flamer-Range01-ScavReduce-Undo", SCAV_6);
+		completeResearch("R-Wpn-Flamer-Range01-ScavReduce-Undo", SCAV_7);
+	}
+	else (difficulty === INSANE)
```
Comment (KJeff01) [inline script/campaign/cam1b.js:93]: else if (difficulty === INSANE)

**Pull request merged**